addons/cluster-autoscaler/README.md

```diff
@@ -1,18 +1,20 @@
 # Cluster Autoscaler Addon
 
-Note that you likely want to change `AWS_REGION` and `GROUP_NAME`, and probably `MIN_NODES` and `MAX_NODES`
+We strongly recommend using Cluster Autoscaler with the kubernetes version for which it was meant. Refer to the [Cluster Autoscaler documentation compatibility matrix]( https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/README.md#releases)
+
+Note that you likely want to change `AWS_REGION` and `GROUP_NAME`, and probably `MIN_NODES` and `MAX_NODES`. Here is an example of how you may wish to do so:
 
 ```bash
 CLOUD_PROVIDER=aws
-IMAGE=gcr.io/google_containers/cluster-autoscaler:v0.6.0
+IMAGE=gcr.io/cluster-autoscaler:v1.1.0
```

Comment:
wrong image, should be `gcr.io/google_containers/cluster-autoscaler:v1.1.0`

Reply:
Fixed and rebased.